tools/xcms_refine/abims_xcms_refine.xml

```diff
@@ -0,0 +1,265 @@
+
+    <macros>
+        <import>macros.xml</import>
+        <import>macros_xcms.xml</import>
+    </macros>
+
+    <expand macro="requirements"/>
+
+    <expand macro="stdio"/>
+
+    <command><![CDATA[
+        @COMMAND_RSCRIPT@/xcms_refine.r
+        image '$image'
+
+        method $methods.method
+        #if $methods.method == "CleanPeaks":
+            maxPeakwidth $methods.maxPeakwidth
+        #elif $methods.method == "FilterIntensity":
+            threshold $methods.threshold
+            nValues $methods.nValues
+            value $methods.value
+        #else:
+            expandRt $methods.expandRt
+            expandMz $methods.expandMz
+            ppm $methods.ppm
+            minProp $methods.minProp
+        #end if
+
+        msLevel $msLevel
+
+        @COMMAND_FILE_LOAD@
+
+        @COMMAND_LOG_EXIT@
+   ]]></command>
+
+    <inputs>
+        <param name="image" type="data" format="rdata.xcms.findchrompeaks" label="@INPUT_IMAGE_LABEL@" help="@INPUT_IMAGE_HELP@ from: findChromPeaks" />
+        <conditional name="methods">
+            <param name="method" type="select" label="Method for peak refinement" help="See the help section below">
+                <option value="CleanPeaks" selected="true">CleanPeaks - remove peaks that are too wide</option>
+                <option value="FilterIntensity">FilterIntensity - remove peaks with too low intensity</option>
+                <option value="MergeNeighboringPeaks">MergeNeighboringPeaks - combine peaks that are too close together</option>
+            </param>
+            <when value="CleanPeaks">
+                <param argument="maxPeakwidth" type="float" value="10" label="MaxPeakWidth" help="maximum peak width in seconds" />
+            </when>
+            <when value="FilterIntensity">
+                <param argument="threshold" type="float" value="0" label="minimal required intensity" help="minimal required intensity for a peak to be retained.  Defaults to 0.0." />
+                <param argument="nValues" type="integer" value="1" label="minimum number of data points per peak" help="number of data points (per chromatographic peak) that have to be >= threshold. Defaults to 1." />
+                <param argument="value" type="text" value="maxo" label="chromatographic intensity type ('intb', 'into', or 'maxo')" help="intensity type (which must be set in the chromatographic peaks input data) that should be used for the filtering. Defaults 'maxo'." />
+                <param name="value" type="select" label="Definition of peak intensity" help="See the manual. 'intb' requires centWave processed data">
+                    <option value="maxo" selected="true">maxo - use maximum peak intensities</option>
+                    <option value="into">into - use integrated peak intensities</option>
+                    <option value="intb">intb - use baseline-corrected integrated peak intensities</option>
+                </param>
```

Comment:
It seems that you have two param with the same name `value`

Reply:
I will investigate

Reply:
I missed that. Apparently, the last param among those named is the one rendered; my good luck therefore was bad luck...  I will fix this.